easybuild/easyblocks/m/molpro.py

```diff
@@ -227,7 +236,14 @@ def install_step(self):
             # put original LAUNCHER definition back in place in bin/molpro that got installed,
             # since the value used during installation point to temporary files
             molpro_path = os.path.join(self.full_prefix, 'bin', 'molpro')
-            apply_regex_substitutions(molpro_path, [(r"^(LAUNCHER\s*=\s*).*$", r"\1 %s" % self.orig_launcher)])
+
+        # Make sure that the launcher text we replace (where this is done) is not
+        # the instance of setting LAUNCHER that contains the eval.
+        launchertext = r"^(LAUNCHER\s*=\s*)((?!\"`eval echo \$LAUNCHER` \$MOLPRO_OPTIONS\").)*$"
+        if self.cfg['parallel_launcher'] is not None:
+            apply_regex_substitutions(molpro_path, [(launchertext, r'\1"%s"' % self.cfg['parallel_launcher'])])
+        elif self.orig_launcher is not None:
+            apply_regex_substitutions(molpro_path, [(launchertext, r"\1%s" % self.orig_launcher)])
```

Comment:
you dropped a space before the `%s`, not sure how space-sensitive things are, but better leave it in place? same above